docs/resources/aviatrix_edge_neo_ha.md

```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_edge_neo_ha"
+description: |-
+  Creates an Aviatrix Edge NEO HA
+---
+
+# aviatrix_edge_neo_ha
+
+The **aviatrix_edge_neo_ha** resource creates the Aviatrix Edge NEO HA.
```

Comment:
Are we to assume that in order to use this resource, we need the (primary) **aviatrix_edge_neo** first? If thats the case, we should make a note of that